UI/obs-app.cpp

```diff
@@ -1108,6 +1108,12 @@ OBSApp::OBSApp(int &argc, char **argv, profiler_name_store_t *store)
 	sleepInhibitor = os_inhibit_sleep_create("OBS Video/audio");
 
 	setWindowIcon(QIcon::fromTheme("obs", QIcon(":/res/images/obs.png")));
+
+#if defined(__linux__)
+	QProcess process;
+	process.start("pkexec modprobe v4l2loopback");
+	process.waitForFinished();
```

Comment:
Would not this make OBS unusable by users lacking admin privileges?

Also users might like to provide options such as device number and name.

For now I would have left to be done by users, and just documented. Ideally some configuration item with those options available to be specified.

Reply:
This is the piece of code in the original plugin which seemed to deal with config etc https://github.com/CatxFish/obs-v4l2sink/blob/master/src/v4l2sinkproperties.cpp#L21

Reply:
I can confirm that this triggers a password dialog for elevation on each launch.

Reply:
If there absolutely must be a password prompt, it should only be when the virtual camera is activated, and not on startup.

Reply:
Done.